aspnetcore/security/app-secrets.md

```diff
@@ -166,6 +166,26 @@ The Secret Manager tool can be used from other directories too. Use the `--proje
 dotnet user-secrets set "Movies:ServiceApiKey" "12345" --project "C:\apps\WebApp1\src\WebApp1"
 ```
 
+> [!TIP]
```

Comment:
We frown on Tips and other chrome. Can we remove TIP chrome and just have the info?